### PR TITLE
Text Input

### PR DESCRIPTION
**Pull request by lduffy-voysis**
- Added sendTextQuery method to Service to allow user to process text requests. 

- Updated SocketRequest to allow for text query. 

- Refactored ServiceImpl and Converter. 

- Updated tests.

### CODE REVIEW

File: Voysis/voysis/Converter.swift [hunk 1]
```diff
@@ -11,7 +11,21 @@ internal struct Converter {
     static let response = "response"
     static let vadStop = "vad_stop"
 
-    static func encodeRequest<T>(socketRequest: SocketRequest<T>) throws -> String? {
+    static func encodeRequest<C: Context>(text: String? = nil, context: C?, userId : String?, token : String) throws -> String? {
```
Comment (jfarrelly-voysis) [inline Voysis/voysis/Converter.swift]: If all the functionality is in here now, can we delete `Converter`?

Comment (lduffy-voysis) [reply Voysis/voysis/Converter.swift]: Yeah I think so, You mean declare the converter functions as top level functions for the converter file and delete the converter class?

File: Voysis/models/request/SocketRequest.swift [hunk 2]
```diff
@@ -32,6 +41,14 @@ public struct AudioQuery: Codable {
     }
 }
 
+public struct TextQuery: Codable {
```
[the remark this answers is not in the document]
Comment (lduffy-voysis) [reply Voysis/models/request/SocketRequest.swift]: Yes it should. Replaced.

File: Voysis/voysis/Converter.swift (hunk 1 above)
Comment (lduffy-voysis) [reply Voysis/voysis/Converter.swift]: Removed the Converter class. Using Converter functions as high level functions.

Comment (jfarrelly-voysis) [reply Voysis/voysis/Converter.swift]: Actually, for some reason I thought this code was in `ServiceImpl` - I don't know what I was thinking.  I think it's actually clearer with the `Converter` class around it.  Sorry about the confusion.

Comment (lduffy-voysis) [reply Voysis/voysis/Converter.swift]: No problem. Reverted.

**Pull request merged**